### ISSUE TITLE
ranger crashed

### ISSUE THREAD

**Issue by anijoekamehameha** [ranger/ranger#2131]
#### Runtime Environment

- Operating system and version: Catalina 10.15.7
- Terminal emulator and version: item2 zsh
- Python version: 3.8.6
- Ranger version/commit: used     Plug 'francoiscabrol/ranger.vim' today

- Locale: 

#### Current Behavior

Crashed
How has this issue affected you?  What are you trying to accomplish? -->

#### Steps to reproduce
1. vim compiled 8.2
2. add plugin with Plug
3. :Ranger

#### Traceback

```

ranger version: ranger-master
Python version: 3.8.6 (default, Oct  8 2020, 14:06:32) [Clang 12.0.0 (clang-1200.0.32.2)]
Locale: None.UTF-8

Traceback (most recent call last):
  File "/usr/local/lib/python3.8/site-packages/ranger/core/main.py", line 173, in main
    fm.initialize()
  File "/usr/local/lib/python3.8/site-packages/ranger/core/fm.py", line 103, in initialize
    set_image_displayer()
  File "/usr/local/lib/python3.8/site-packages/ranger/core/fm.py", line 102, in set_image_displayer
    self.image_displayer = get_image_displayer(self.settings.preview_images_method)
  File "/usr/local/lib/python3.8/site-packages/ranger/ext/img_display.py", line 100, in get_image_displayer
    image_displayer_class = IMAGE_DISPLAYER_REGISTRY[registry_key]
  File "/usr/local/lib/python3.8/site-packages/ranger/ext/img_display.py", line 80, in fallback_image_displayer
    raise ImgDisplayUnsupportedException
ranger.ext.img_display.ImgDisplayUnsupportedException

#### Expected Behavior

#### Context
<!-- 
```

**Comment by toonn** [comment]
Hmm, image previews work for you outside of vim?

**Comment by anijoekamehameha** [comment]
nop

**Comment by toonn** [comment]
`ranger --clean` then `:set preview_images_method iterm2` and `:set preview_images True` and check if previews work.

**Comment by anijoekamehameha** [comment]
all tried. Not working sir.

Thanks
Regards

**Comment by anijoekamehameha** [comment]
but will try again

**Comment by anijoekamehameha** [comment]
tried and getting this

─ ~                                                         ✔  06:48:57
PM
╰─ ranger --clean
zsh: command not found: ranger

╭─ ~                                                     127 ✘  06:49:07
PM
╰─ brew install ranger
Updating Homebrew...
==> Auto-updated Homebrew!
Updated 2 taps (homebrew/core and homebrew/cask).
==> Updated Formulae
Updated 10 formulae.
==> Updated Casks
origin       piezo        reaper       syncplay     torguard     usenapp

Warning: ranger 1.9.3 is already installed, it's just not linked
You can use `brew link ranger` to link this version.

╭─ ~                                                   ✔  19s  06:49:35
PM
╰─ brew link ranger
Linking /usr/local/Cellar/ranger/1.9.3...
Error: Could not symlink share/doc/ranger/examples/README
Target /usr/local/share/doc/ranger/examples/README
already exists. You may want to remove it:
  rm '/usr/local/share/doc/ranger/examples/README'

To force the link and overwrite all conflicting files:
  brew link --overwrite ranger

To list all files that would be deleted:
  brew link --overwrite --dry-run ranger

╭─ ~                                                       1 ✘  06:49:43
PM
╰─ sudo rm  /usr/local/share/doc/ranger/examples/README
Password:

╭─ ~                                                    ✔  5s  06:50:30
PM
╰─ brew link --overwrite ranger
Linking /usr/local/Cellar/ranger/1.9.3...
Error: Could not symlink share/doc/ranger/examples/README
/usr/local/share/doc/ranger/examples is not writable.

**Comment by toonn** [comment]
No need to brew link for ranger. Sounds like an issue with your PATH.

**Comment by anijoekamehameha** [comment]
ohhk. Will try. Or will stuck to nerd tree. But I really loved ranger. If
you want me to to follow clean instructions, will be happy.  It was working
really fine till yesterday.
Thanks
Regards

**Comment by anijoekamehameha** [comment]
please share clean instructions.

**Comment by toonn** [comment]
Same instructions from before. We're just trying to figure out whether ranger works properly without the vim plugin first.

**Comment by anijoekamehameha** [comment]
sure. Please let me know when it works as I have recommend ranger to many.
Thanks
Regards

**Comment by anijoekamehameha** [comment]
without vim. Sorry I forgot to say.
Regards

**Comment by toonn** [comment]
No, *you* need to execute the steps I told you so *we* can figure out the problem together. I'm using ranger without any problems on macOS so there must be something wrong with your setup.

**Comment by anijoekamehameha** [comment]
OK. Let me try again tomorrow. Will get back to you.
Thanks
Regards

**Comment by anijoekamehameha** [comment]
Hello
1. I removed the ranger completely first.
2. then installed with brew install ranger
3. gave symlink error. it couldn't symlink
4. but its running directly when i run from the directory directly
5. Please find the screenshot of image preview after I set it to Iterm as
you suggested. Its not showing image but giving info about an image.

Please suggest anything else I can try

Thanks a ton for your help.

Regards

**Comment by anijoekamehameha** [comment]
its working now now. Its showing previews in current try.
it works only with --clean.
no luck yet in vim

thanks
regards

**Comment by toonn** [comment]
If it works with `--clean` there's something wrong with your `rc.conf` or your `scope.sh`, so you'll need to figure that out. Maybe start from a fresh copy and do your modifications one by one.

**Comment by anijoekamehameha** [comment]
yep. Will try that. Thanks.
Regards
